test/integration/storage/test_buckets.rb

```diff
@@ -1,55 +1,89 @@
 require "helpers/integration_test_helper"
+require "integration/storage/storage_shared"
+require "securerandom"
+require "base64"
+require "tempfile"
 
-class TestBuckets < FogIntegrationTest
-  begin
-    @@connection = Fog::Storage::Google.new
-    @@connection.delete_bucket("fog-smoke-test")
-  rescue Exception => e
-    # puts e
+class TestStorageRequests < StorageShared
+  def test_put_bucket
+    sleep(1)
```

Comment:
Bucket operations are [strictly rate limited](https://cloud.google.com/storage/docs/best-practices#buckets-object). Is there a more idiomatic option available to rate limit tests?